mono/metadata/appdomain.h

```diff
@@ -145,7 +145,7 @@ MONO_API int32_t
 mono_context_get_domain_id  (MonoAppContext *context);
 
 MONO_API MonoJitInfo *
-mono_jit_info_table_find   (MonoDomain *domain, char *addr);
+mono_jit_info_table_find   (MonoDomain *domain, void* addr);
```

Comment:
this is part of the API, afaik we aren't supposed to change that.

Reply:
How could this break anything?
The only thing it could break, a large stretch of the imagination, is if you use C++ decltype to duplicate the type and you depend on the name mangling being stable across versions. Our own name mangling would not change since we are extern "C".